toolkit/Makefile

```diff
@@ -19,16 +21,29 @@ PACKAGE_REBUILD_LIST    ?=
 PACKAGE_IGNORE_LIST     ?=
 SRPM_PACK_LIST          ?=
 
-REBUILD_TOOLCHAIN               ?= n
+# Logic to auto configure build options for optimized builds
+# These will set the default values for: REBUILD_TOOLS, REBUILD_TOOLCHAIN, DELTA_BUILD, INCREMENTAL_TOOLCHAIN, and ALLOW_TOOLCHAIN_DOWNLOAD_FAIL
+include ./quickrebuild_autoconfig.mk
+
+# We generally want to use the DELTA_BUILD flag to determine whether to use the incremental toolchain, but to preserve
+# backwards compatibility, we'll still use the INCREMENTAL_TOOLCHAIN flag if it's set.
+DELTA_BUILD                     ?= n
+ifeq ($(origin DELTA_BUILD), undefined)
+# We didn't set DELTA_BUILD, so use the previous default behavior of INCREMENTAL_TOOLCHAIN for backwards compatibility.
 INCREMENTAL_TOOLCHAIN           ?= n
+else
+# We set DELTA_BUILD, so inherit value for INCREMENTAL_TOOLCHAIN.
+INCREMENTAL_TOOLCHAIN           ?= $(DELTA_BUILD)
```

Comment:
Thank you

Reply:
Any chance we can echo a message that "INCREMENTAL_TOOLCHAIN" is deprecated?

So we can yank it out in the future.  No reason to have both